tests/tt_metal/tt_metal/unit_tests/compute/test_broadcast.cpp

```diff
@@ -289,11 +289,11 @@ void run_single_core_broadcast(tt_metal::Device* device, const BroadcastConfig&
 }
 }
 
-class BroadcastParametrizedDeviceFixture : public DeviceFixture,
+class BroadcastParameterizedDeviceFixture : public DeviceFixture,
```

Comment:
currently everything in `unit_tests/compute` are meant to be simple single core LLK test. I think we should have a dedicated LLK or Compute fixture for all the tests in this directory that currently run using `DeviceFixture`